JapaneseFilter/sections/specific.txt

```diff
@@ -3,6 +3,7 @@
 !
 ! /banner/banner_ - if it is necessary, make this rule domain specific
 !
+auctions.yahoo.co.jp##.ss
```

Comment:
I don't see it. Please check it.

Reply:
Probably they're looking geo location etc.

![yahoo](https://user-images.githubusercontent.com/58900598/91029067-45fdb280-e639-11ea-9342-bf494569feec.png)